examples/README.md

```diff
@@ -132,6 +132,7 @@ Example | Description
 [Update glTF Scene](../examples/3d/update_gltf_scene.rs) | Update a scene from a glTF file, either by spawning the scene as a child of another entity, or by accessing the entities of the scene
 [Vertex Colors](../examples/3d/vertex_colors.rs) | Shows the use of vertex colors
 [Wireframe](../examples/3d/wireframe.rs) | Showcases wireframe rendering
+[tonemapping](../examples/3d/tonemapping.rs) | Compares tonemapping options
```

Comment:
```suggestion
[Tonemapping](../examples/3d/tonemapping.rs) | Compares tonemapping options
```

Capitalization

Reply:
This file is auto generated, you need to modify the Cargo.toml example metadata, and then regenerate it.